### PR TITLE
Added division example to poster

### PR DESCRIPTION
**Pull request by jorisburgers**

### CODE REVIEW

**Comment by joristt** [comment]
Should we explicitly mention that the third example is an identity function?

File: poster/poster.tex [hunk 1]
```diff
@@ -77,9 +77,28 @@
...
+		\end{minipage}
+		\vspace{1cm}
+		\item \textsc{\small Reverse of division}\\ 			\textsc{\tiny{automatically computed}} \\ \vspace{1cm}
+		\begin{minipage}{.8\textwidth}
+		\code{divide_reverse}
+		\end{minipage}
 	\end{itemize}
+	\begin{itemize}
+\item Procedures are called with either the \textit{call} or the \textit{uncall} keyword. The \textit{main} procedure is called automatically. 	
+		\item
```
Comment (omelkonian) [inline poster/poster.tex]: remove this `'item`

File: poster/poster.tex [hunk 2]
```diff
@@ -77,9 +77,28 @@
...
-		\item Show inverse side-by-side
+
+		\item The inverse of a program can be computed automatically.
+		\center
+		\textsc{\small Division}\\ \vspace{1cm}
+		\begin{minipage}{.8\textwidth}
+		\code{divide}
+		\end{minipage}
+		\vspace{1cm}
+		\item \textsc{\small Reverse of division}\\ 			\textsc{\tiny{automatically computed}} \\ \vspace{1cm}
```
Comment (omelkonian) [inline poster/poster.tex]: Better to not repeat that its automatically computed (you mention it in the text above. If you'd like, put `(for free!)` next to `\small Reverse of division`

**Comment by omelkonian** [comment]
@joristt nah, let's leave the audience figure it out :dagger:

**Comment by jorisburgers** [comment]
@joristt I also don't think we should mention it on the poster, we could always explain it when people ask about it.

**Pull request merged**